### PR TITLE
Add refresh button to office queue

### PR DESCRIPTION
**Pull request by garrettqmartin8**
## Description

As office user
I want to be able to refresh my queue
So that I see latest information without reloading whole application

## Setup

1. `make server_run`
2. `make client_run`

## Code Review Verification Steps

* [x] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [x] User facing changes have been reviewed by design.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/165770552) for this change

## Screenshots

refresh_queue

### CODE REVIEW

File: src/scenes/Office/QueueTable.jsx [hunk 1]
```diff
@@ -94,6 +106,12 @@ class QueueTable extends Component {
       }
     });
 
+    let refreshIconClassList = ['refresh'];
```
Comment (Igarfinkle) [inline src/scenes/Office/QueueTable.jsx]: An alternative to these lines here would be to just have the classname below be 

`className={'refresh' + this.state.refreshing ? 'focused' : ''}`

this solution here uses fewer lines, but I'll leave it up to you whether it's more elegant/robust.

**Comment by Igarfinkle** [comment]
Also, probably worth making sure that the dotted-line border is intentional and not just an artifact from design.

File: src/scenes/Office/QueueTable.jsx (hunk 1 above)
Comment (garrettqmartin8) [reply src/scenes/Office/QueueTable.jsx]: Good call, just replace it with the inline version you suggested.

**Comment by garrettqmartin8** [comment]
@Igarfinkle, I made the adjustment you suggested and I'm also confirming the dotted border with Jenny in design review.

**Pull request merged**